fatiando/seismic/conv.py

```diff
@@ -0,0 +1,251 @@
+"""
+Zero-offset convolutional seismic modeling
+
+Give a depth model and obtain a seismic zero-offset convolutional gather. You
+can give the wavelet, if you already have, or use one of the existing, from
+which we advise ricker wavelet (rickerwave function).
+
+* :func:`~fatiando.seismic.conv.convolutional_model`: given the reflectivity
+  series and wavelet, it returns the convolutional seismic gather.
+* :func:`~fatiando.seismic.conv.reflectivity`: calculates the reflectivity
+  series from the velocity model (and density model if present).
+* :func:`~fatiando.seismic.conv.depth_2_time`: convert depth property model to
+  the model in time.
+* :func:`~fatiando.seismic.conv.rickerwave`: calculates a ricker wavelet.
+
+Examples
+--------
+.. plot::
+    :include-source:
+    :context:
+
+    >>> import numpy as np
+    >>> import matplotlib.pyplot as plt
+    >>> from fatiando.seismic import conv
+    >>> from fatiando.vis import mpl
+    >>> # Choose some velocity depth model
+    >>> n_samples, n_traces = [600, 20]
+    >>> rock_grid = 1500.*np.ones((n_samples, n_traces))
+    >>> rock_grid[300:, :] = 2500.
+    >>> # Convert from depth to time
+    >>> [vel_l, rho_l] = conv.depth_2_time(rock_grid, dt=2.e-3)
+    >>> # Calculate the reflectivity for all the model
+    >>> rc = conv.reflectivity(vel_l,rho_l)
+    >>> # Convolve the reflectivity with a ricker wavelet
+    >>> synt = conv.convolutional_model(rc, 30., conv.rickerwave)
+    >>> # Plot the result
+    >>> fig = plt.figure(figsize=(6,5))
+    >>> _ = mpl.seismic_wiggle(synt, dt=2.e-3)
+    >>> _ = mpl.seismic_image(synt, dt=2.e-3,
+    ...                            cmap=mpl.pyplot.cm.jet, aspect='auto')
+    >>> _ = plt.ylabel('time (seconds)')
+    >>> _ = plt.title("Convolutional seismogram", fontsize=13, family='sans-serif')
+
+References
+----------
+
+Yilmaz, Oz,
+Ch.2 Deconvolution. In: YILMAZ, Oz. Seismic Data Analysis: Processing,
+Inversion, and Interpretation of Seismic Data. Tulsa: Seg, 2001. Cap. 2.
+p. 159-270. Available at: <http://dx.doi.org/10.1190/1.9781560801580.ch2>
+
+
+"""
+from __future__ import division
+import numpy as np
+from scipy import interpolate  # linear interpolation of velocity/density
+
+
+def convolutional_model(rc, f, wavelet, dt=2.e-3):
```

Comment:
@victortxa now that I'm thinking about this better, I don't think `dt` should have a default value. This is the sampling interval of `rc` and the results would be totally wrong if this value is wrong. So it's better to require the user to think about.